src/components/interaction-designer/block-editors/LabelEditor.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="block-label">
+  <div class="block-label" :id="`${block.uuid}.label`">
```

Comment:
Shouldn't we add a `/` after `${block.uuid}` according to AC ?

Reply:
As per AC we want the URL to be in `{blockId}/.config.choices[1].audio` format. I did not want to have a mixed set (/ and .) which is why I chose to use . instead of /. I will think more on this while implementing [VMO-3478](https://viamoinc.atlassian.net/browse/VMO-3478). Does that sound good ?

Reply:
Sounds good